web/src/components/Plugin/data.tsx

```diff
@@ -74,6 +74,15 @@ export enum PluginType {
   other = 'other',
 }
 
+export enum PluginState {
+  disable = 10,
+  enable = 20,
+}
+
+export interface PluginItem extends PluginComponent.Meta {
+  state?: PluginState;
+}
```

Comment:
If it is a writing type, we'd better put it in the type file.

Reply:
The real code cannot be called if it is placed in the type file, it is written here to make it easier for the code in JSX to use